docs/system-requirements/software/perl.rst

```diff
@@ -74,7 +74,11 @@ To force the install for any reason, use the "-f" flag, so type:
 
   $ cpan -f Module::Name
 
-Don't forget to use sudo in front of the CPAN command if you have not set the sudo option in the CPAN configuration
+If you have not set the sudo option in the CPAN configuration,Add sudo before the CPAN command, like:
+
+.. code-block:: bash
+
+  $ sudo cpan -f Module::Name
 
```

Comment:
I don't think that the `-f` flag is required unless you have some good reasons to use it, plus this instruction is general not just for the force install section